src/removeNode.ts

```diff
@@ -10,6 +10,7 @@ export async function handler(event: OldAndNewNodeResponse): Promise<OldAndNewNo
     const newestInstance: Instance = event.newestElasticsearchNode.ec2Instance;
 
     if (event.oldestElasticsearchNode.isMasterEligible) try {
+        console.log(`attempting to terminate oldestInstance: ${oldestInstance.id}`);
```

Comment:
As 'terminate' has a specific meaning in AWS EC2 jargon, best to avoid that term here:

```suggestion
        console.log(`attempting to smoothly shutdown master-eligible node: ${oldestInstance.id}`);
```